extensions/rescuetime-focus-session-trigger/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# RescueTime Focus Session Trigger Changelog
+
+## Added RescueTime Focus Session Trigger - (2022-3-19)
```

Comment:
```suggestion
## [Added RescueTime Focus Session Trigger] - 2022-3-19
```
New format in the [changelog](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history)